seedsource_core/django/seedsource/management/commands/create_vector_tiles.py

```diff
@@ -0,0 +1,89 @@
+from django.core.management import BaseCommand
+from django.conf import settings
+from seedsource_core.django.seedsource.models import SeedZone
+from tempfile import mkdtemp
+import os
+import subprocess
+import json
+import shutil
+
+
+class Command(BaseCommand):
+    help = 'Facilitates converting of vector data into vector tiles.'
+
+    def _write_out(self, output):
+        self.stdout.write('\033[0;33m' + output + '\033[0m')
+
+    def _create_folder(self, directory):
```

Comment:
Given that this is only two lines and only used once, I think it makes more sense to simply inline the code. It's actually more code overall this way.